src/main/java/ByteTalkerException.java

```diff
@@ -0,0 +1,19 @@
+public class ByteTalkerException {
+    public static class TODONoTaskException extends Exception {
```

Comment:
The naming convention should not be in uppercase. Only constants should have uppercase. 

```suggestion
    public static class TodoNoTaskException extends Exception {
```
Same issue for DEADLINEException